### ISSUE TITLE
Selected state is not announced to screen readers

### ISSUE THREAD

**Issue by geoffrich** [AlaskaAirlines/auro-pane#12]
## Describe the bug
When navigating to a selected pane with a screen reader, there is no indication that the pane is selected.

## To Reproduce
1. Go to the pane [demo page](https://auro.alaskaair.com/components/auro/pane/)
2. Navigate to a selected pane with a screen reader
3. Notice that the selected state is not read out

## Expected behavior
There is an indication that the pane is in a selected state. This could be accomplished with visually hidden text.

## Desktop (please complete the following information):
- Browser: Firefox 
- Screen reader: NVDA

## Additional context
This is in the context of the disruptions center, where we display a carousel of panes with different dates. When browsing the carousel on the initial load, there is no way to tell which pane is selected if you can't see the visual difference.

We could control this ourselves by adding visually hidden text to the slot, but I wondered if it would make sense to have this baked into the component.

## Additional, additional context...

It has been some time since the code of this element has been reviewed, there will need to be some reacquaintance with how auro-pane interacts with auro-carousel and how the select state is managed. 

Post this review, this issue may need to be revised as to how a possible solution will work. Is it the responsibility of the carousel or pane elements? 

Any radical refactoring of the carousel or pane elements is not warranted in the scope of this work without a conversation with the team. 

## Exit criteria

As a visually disabled user, I expect to hear the selected pane in the carousel based on the shoulder-date experience. 

It is assumed that since the date content is coming from date.js, the selected screen readers will speak out the date in a human-sounding phrase.

**Comment by blackfalcon** [comment]
Moving this to `ready to work`. Must have missed that in refinement.

**Comment by blackfalcon** [comment]
Moving this BACK to `refinement` as there is a request to discuss how to implement this change regarding `checkbox` type functionality.

**Comment by blackfalcon** [comment]
@jordanjones243 and @jason-capsule42 had discussed options about making this element more `radio-button` link in functionality. 

Meeting has been scheduled with the dev team.

**Comment by blackfalcon** [comment]
Outcome of meeting was not captured in this issue. 

It was decided NOT to refactor this element as suggested and use the existing features within the element to address this accessibility issue.

**Comment by blackfalcon** [comment]
This issue will be closed in favor of the following issue with specified outcomes. 

- https://github.com/AlaskaAirlines/auro-pane/issues/32
